neqo-transport/src/frame.rs

```diff
@@ -267,23 +267,26 @@ impl Frame {
         if offset > 0 {
             remaining = remaining.saturating_sub(Encoder::varint_len(offset));
         }
-        let (fin, fill) = if data.len() > remaining {
-            if remaining == 0 {
-                return None;
+        let (fin, fill) = match data.len().cmp(&remaining) {
```

Comment:
This is going to conflict with #370.

Reply:
Let's wait until #370 lands then so I can rebase :)